### DISCUSSION TITLE
Mod selection menu changes abruptly as it opens

### DISCUSSION THREAD

**Discussion by Flamiii** [ppy/osu#16789] [Q&A] [answered]
https://user-images.githubusercontent.com/74088315/152653806-e9fc3dc1-8b73-41bd-a5f8-4beb8e03dda0.mp4

This might seem kind of nitpicky but I don't think that this is intentional and it just feels kind of weird.

**Comment by bdach** [comment, marked as answer]
known issue, and menu in question is pending a redesign sooner rather than later. not sure it's worth fixing.

**Reply by peppy** [reply]
it's definitely not worth fixing. a fix was applied in the past and was unapplied due to it breaking expectations in other ways.
